src/material-experimental/mdc-slider/slider.spec.ts

```diff
@@ -0,0 +1,1449 @@
+    it('should increment slider by 4 on page up pressed', () => {
+      expect(testComponent.onChange).not.toHaveBeenCalled();
+
+      dispatchKeyboardEvent(sliderNativeElement, 'keydown', PAGE_UP);
+      fixture.detectChanges();
+
+      // The `onInput` event should be emitted once due to a single keyboard press.
+      expect(testComponent.onInput).toHaveBeenCalledTimes(1);
+      expect(testComponent.onChange).toHaveBeenCalledTimes(1);
+      expect(sliderInstance.value).toBe(4);
+    });
+
+    // TODO(devversion): MDC decrements the slider by "4" on page up. The standard
+    // Material slider decrements by "10".
+    it('should decrement slider by 4 on page down pressed', () => {
+      fixture.componentInstance.value = 100;
+      fixture.detectChanges();
+
+      expect(testComponent.onChange).not.toHaveBeenCalled();
+
+      dispatchKeyboardEvent(sliderNativeElement, 'keydown', PAGE_DOWN);
+      fixture.detectChanges();
+
+      // The `onInput` event should be emitted once due to a single keyboard press.
+      expect(testComponent.onInput).toHaveBeenCalledTimes(1);
+      expect(testComponent.onChange).toHaveBeenCalledTimes(1);
+      expect(sliderInstance.value).toBe(96);
+    });
+
+    it('should set slider to max on end pressed', () => {
+      expect(testComponent.onChange).not.toHaveBeenCalled();
+
+      dispatchKeyboardEvent(sliderNativeElement, 'keydown', END);
+      fixture.detectChanges();
+
+      // The `onInput` event should be emitted once due to a single keyboard press.
+      expect(testComponent.onInput).toHaveBeenCalledTimes(1);
+      expect(testComponent.onChange).toHaveBeenCalledTimes(1);
+      expect(sliderInstance.value).toBe(100);
+    });
+
+    it('should set slider to min on home pressed', () => {
+      fixture.componentInstance.value = 100;
+      fixture.detectChanges();
+
+      expect(testComponent.onChange).not.toHaveBeenCalled();
+
+      dispatchKeyboardEvent(sliderNativeElement, 'keydown', HOME);
+      fixture.detectChanges();
+
+      // The `onInput` event should be emitted once due to a single keyboard press.
+      expect(testComponent.onInput).toHaveBeenCalledTimes(1);
+      expect(testComponent.onChange).toHaveBeenCalledTimes(1);
+      expect(sliderInstance.value).toBe(0);
+    });
+
+    it(`should take no action for presses of keys it doesn't care about`, () => {
+      fixture.componentInstance.value = 50;
+      fixture.detectChanges();
+
+      expect(testComponent.onChange).not.toHaveBeenCalled();
+
+      dispatchKeyboardEvent(sliderNativeElement, 'keydown', BACKSPACE);
+      fixture.detectChanges();
+
+      // The `onInput` event should be emitted once due to a single keyboard press.
+      expect(testComponent.onInput).not.toHaveBeenCalled();
+      expect(testComponent.onChange).not.toHaveBeenCalled();
+      expect(sliderInstance.value).toBe(50);
+    });
+
+    // TODO: MDC slider does not respect modifier keys.
+    // tslint:disable-next-line
```

Comment:
File a bug on their repo?

Reply:
Looks like it's by intention that they don't respect modifier keys. Native range sliders don't seem to respect these either (i.e. value still changes with SHIFT + LEFT ARROW).

How should we proceed? based on their implementation of the keyboard logic we can't overwrite/configure this behavior.